code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -310,3 +310,8 @@
 /obj/item/weapon/storage/box/hug/reverse_revolver/New()
 	..()
 	new /obj/item/weapon/gun/ballistic/revolver/reverse(src)
+
+/obj/item/weapon/storage/box/syndie_kit/mimery/New()
+	..()
+	new /obj/item/weapon/spellbook/oneuse/mimery_one(src)

```

Comment:
can you make the type a bit more descriptive to like `mimery_blockade` and `mimery_gun`?